index.bs

```diff
@@ -5245,16 +5252,33 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
 1. Verify that the [=UP=] bit of the <code>[=flags=]</code> in |authData| is set.
 
-1. If the [=[RP]=] requires [=user verification=] for this assertion,
+1. Determine whether [=user verification=] is required for this assertion.
+    [=User verification=] SHOULD be required if any of the following is true:
+
+    - <code>|options|.{{PublicKeyCredentialRequestOptions/userVerification}}</code> is set to {{UserVerificationRequirement/required}}.
+    - <code>|options|.{{PublicKeyCredentialRequestOptions/userVerification}}</code> is set to {{UserVerificationRequirement/preferred}}
+        and <code>|credentialRecord|.[=credential record/UV=]</code> is [TRUE].
+    - The [=[RP]=] has any other reason to require [=user verification=].
+
+    [=User verification=] SHOULD NOT be required
```

Comment:
Hm, the intent is that if UV is not determined to be required, you simply ignore the value of the UV flag. Is that not clear, then? I guess we could formalize this even further with a Boolean variable for whether UV was determined to be required, but I'd rather not of we can help it.

Reply:
I think the better wording is "the state of UV bool is not checked if the policy is discouraged", or at least something more explicit that says to ignore checking this parameter. I think we should formalise it better because even to me I read it incorrectly as is 😅

Reply:
Does af07d1598c70431d36ee1ad53e042582b72f7379 address this concern?

Reply:
Yes I think it does :)